docs/docs/external/getting-started/test-network/ledger-initialization.md

```diff
@@ -255,7 +255,39 @@ Initialize bond and token asset states and ownerships on the `network1` ledger b
 ```bash
 ./scripts/initAssetsForTransfer.sh
 ```
+The `initAssetsForTranser.sh` script above also fetches the certificate of the users in base64 format and stores in the folder `src/data/remoteNetworkUsers` with filename `network1_UsersAndCerts.json` and `network2_UsersAndCerts.json` corresponding to `network1` and `network2` respectively. These files are referred during pledge to fetch the certificate of the recipient of the transferred asset, and during claim to fetch the certificate of the pledger of the asset. *This is because, asset transfer doesn't require both the users being members of neither in the export nor in the import network. It's just sufficient that pledger is part of the asset export network and claimer is part of the asset import network*.
+  * The user certificate files corresponding to the Corda networks also need to be copied into this folder.
+
+### Initializing the Corda Networks
 
+Once the Corda networks (`Corda_Network` and `Corda_Network2`) are launched, the client applications (built earlier) needs to be exercised to generate ledger state in both exporting/source and importing/destination networks in preparation to test asset transfer interoperation flows.
+
+#### Bootstrapping Networks and Application States
+The Corda network ledger (or _vault_ on each node) must be initialized with access control policies, verification policies, and security group information for the other networks (two Fabric networks and other Corda network).
+
+Bootstrap the Corda networks and application states as follows (_the following instructions will initialize either or both Corda networks, depending on which of those are up and running_):
+- Navigate to the `samples/corda/corda-simple-application` folder.
+- Run the following:
+  * On the node owned by `PartyA` in `Corda_Network`:
```

Comment:
Not sure what "On the node...." means here. The user is just supposed to run these commands after navigating to the right folder, correct? If you want to say something about what the commands are doing, maybe add a small sentence after the commands.